vllm_ascend/worker/model_runner_v1.py

```diff
@@ -1192,13 +1489,31 @@ def execute_model(
             if not scheduler_output.total_num_scheduled_tokens:
                 # Return empty ModelRunnerOuptut if there's no work to do.
                 return EMPTY_MODEL_RUNNER_OUTPUT
-            (attn_metadata, hidden_states, spec_decode_metadata, positions,
-             num_scheduled_tokens,
-             sample_indices) = (self._process_reqs(scheduler_output,
-                                                   intermediate_tensors))
+            
+            attn_metadata_dict, logits_indices, spec_decode_metadata = (
+                self._prepare_inputs(scheduler_output)
```

Comment:
what is the difference between `_process_reqs` and `_prepare_inputs`? Do we need a new `_prepare_inputs` method? Can we  add eagle realated codes to  `_process_reqs`?